ArduPlane/quadplane.cpp

```diff
@@ -328,8 +328,8 @@ const AP_Param::GroupInfo QuadPlane::var_info[] = {
 
     // @Param: OPTIONS
     // @DisplayName: quadplane options
-    // @Description: This provides a set of additional control options for quadplanes. LevelTransition means that the wings should be held level to within LEVEL_ROLL_LIMIT degrees during transition to fixed wing flight, and the vehicle will not use the vertical lift motors to climb during the transition. If AllowFWTakeoff bit is not set then fixed wing takeoff on quadplanes will instead perform a VTOL takeoff. If AllowFWLand bit is not set then fixed wing land on quadplanes will instead perform a VTOL land. If respect takeoff frame is not set the vehicle will interpret all takeoff waypoints as an altitude above the current position. When Use QRTL is set it will replace QLAND with QRTL for failsafe actions when in VTOL modes. When AIRMODE is set AirMode is automatically enabled if arming by RC channel.
-    // @Bitmask: 0:LevelTransition,1:AllowFWTakeoff,2:AllowFWLand,3:Respect takeoff frame types,4:Use a fixed wing approach for VTOL landings,5:Use QRTL instead of QLAND for failsafe when in VTOL modes,6:Use idle governor in MANUAL,7:QAssist force enabled,8:Tailsitter QAssist motors only,9:enable AirMode if arming by aux switch,10:Enable motor tilt when disarmed,11:Delay spoolup for 2 seconds after arming
+    // @Description: This provides a set of additional control options for quadplanes.
+    // @Bitmask: 0:Keep wings within LEVEL_ROLL_LIMIT and only use forward motor(s) for climb during transition,1:Allow fixed wing takeoff for TAKEOFF mission command,2:Allow fixed wing landing for LAND mission command,3:VTOL takeoff altitude is above current position rather than HOME,4:Use Fixed Wing Approach for VTOL landings,5:Use QRTL instead of QLAND for failsafe when in VTOL modes,6:Use Idle Governor in MANUAL for forward motor,7:Force QAssist on always,8:QAssist in tailsitters only uses VTOL motors,9:Enable AirMode if arming by aux switch,10:Enable motor tilt for yaw when disarmed,11:Delay VTOL spoolup for 2 seconds after arming
```

Comment:
Lower case is better, but the description for bit 3 (the respect frame stuff) is now backwards. The default is the bit not set, in which case takeoffs are relative to the *current position* not home, or anything else. If you set the bit, then it follows the item you uploaded to the aircraft. The key point to understand here is if you plan a VTOL takeoff to 200 feet, but are at 400 feet and jump to the takeoff command the plane will takeoff another 200 feet to 600 feet above the ground if the bit isn't set.

Reply:
will fix thanks...